acme/acme/client.py

```diff
@@ -521,10 +523,11 @@ def _wrap_in_jws(self, obj, nonce):
         :rtype: `.JWS`
 
         """
-        jobj = obj.json_dumps().encode()
-        logger.debug('Serialized JSON: %s', jobj)
+        jobj = obj.json_dumps(indent=2).encode()
+        logger.debug('JWS payload:\n%s', jobj)
```

Comment:
for extra fancy points: `os.linesep`

we do use both in the codebase though so feel free to ignore this.